.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+package-lock.json*
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+yarn.lock*
```

Comment:
I'd better leave yarn.lock. It works fine so there's no need to keep it out of working tree